test/src/main/java/org/apache/accumulo/test/AuditMessageIT.java

```diff
@@ -324,17 +313,19 @@ public void testImportExportOperationsAudits() throws AccumuloSecurityException,
 
     // Prepare to export the table
     Path confDirPath = getCluster().getConfig().getDir().toPath();
-    File exportDir = confDirPath.resolve("export").toFile();
-    File exportDirBulk = confDirPath.resolve("export_bulk").toFile();
-    assertTrue(exportDirBulk.mkdir() || exportDirBulk.isDirectory());
+    Path exportDir = confDirPath.resolve("export");
+    Path exportDirBulk = confDirPath.resolve("export_bulk");
+    if (!Files.isDirectory(exportDirBulk)) {
+      Files.createDirectories(exportDirBulk);
+    }
```

Comment:
could add similar assertion: `assertTrue(Files.isDirectory(dir))` or move to util method I mentioned earlier